docs/install/_index.md

```diff
@@ -8,11 +8,11 @@ aliases:
   - /docs/getting-started
 ---
 
-You can install both [Redis](https://redis.io/docs/about/) or [Redis Stack](/docs/about/about-stack) locally on your machine. Redis and Redis Stack are available on Linux, macOS, and Windows. 
+You can install both [Redis](https://redis.io/docs/about/) and [Redis Stack](/docs/about/about-stack) locally on your machine. Redis and Redis Stack are available on Linux, macOS, and Windows. 
```

Comment:
Hi Enoch. Thank you for your contribution!

Given that the two installation streams really are separate in the docs, and, leaving off the fact that you can install Redis Stack modules on an existing Redis server, I prefer this:

```suggestion
You can install [Redis](https://redis.io/docs/about/) or [Redis Stack](/docs/about/about-stack) locally on your machine. Redis and Redis Stack are available on Linux, macOS, and Windows. 
```

Reply:
Hi, David. This is actually better than what I suggested, thanks for pointing this out.